contrib/kind.sh

```diff
@@ -115,6 +116,7 @@ usage() {
     echo "-ho  | --hybrid-enabled             Enable hybrid overlay. DEFAULT: Disabled."
     echo "-ds  | --disable-snat-multiple-gws  Disable SNAT for multiple gws. DEFAULT: Disabled."
     echo "-dp  | --disable-pkt-mtu-check      Disable checking packet size greater than MTU. Default: Disabled"
+    echo "-df  | --disable-forwarding         Disable forwarding on OVNK managed interfaces. Default: Enabled"
```

Comment:
isn't `disable-forwarding` [disabled](https://github.com/ovn-org/ovn-kubernetes/pull/3374/files#diff-74a1a8552feeb3203caa0b2f9cc5c9c0eeca76451e37f1a61865c598b3247444R474) by default?

Reply:
Lol true. That is a bit confusing.. the forwarding is enabked by default haha.. disable is disabled by default..

Reply:
fixed this.